optimism/src/mips/interpreter.rs

```diff
@@ -642,15 +697,24 @@ mod tests {
         let mut rng = rand::thread_rng();
         let dummy_preimage_oracle = PreImageOracle::create(&host_program);
         Env {
-            instruction_parts: InstructionParts::default(),
             instruction_counter: 0,
-            // Only 4kb of memory (one PAGE_ADDRESS_SIZE)
-            memory: vec![(0, vec![rng.gen(); PAGE_SIZE as usize])],
-            memory_write_index: vec![(0, vec![0; PAGE_SIZE as usize])],
+            // Only 8kb of memory (two PAGE_ADDRESS_SIZE)
```

Comment:
why is it now twice as much?

Reply:
oh one is for read write and the other is executable memory?

what exactly is executable memory by the way?

Reply:
I needed somewhere to store the opcode that we're going to execute, so I invented an extra page of 'executable memory' to store it.